src/frontend-samples/cross-probing-sample/CrossProbingApp.tsx

```diff
@@ -2,113 +2,57 @@
 * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
-
-import "common/samples-common.scss";
-import { IModelApp, IModelConnection, SelectionSetEvent, SelectionSetEventType, ViewCreator2d, Viewport } from "@bentley/imodeljs-frontend";
+import * as React from "react";

```

Comment:
Try reproducing this error now. I've seen it before, but i can't reproduce it locally. I've made changes to the cross-probing sample so it might be fixed.

Reply:
can no longer reproduce